app/controllers/stories_controller.rb

```diff
@@ -0,0 +1,40 @@
+class StoriesController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  def index
+    @stories = Story.all
+    render json: @stories
+  end
+
+  def show
+    @story = Story.find(params[:id])
+    render json: @story
+  end
+
+  def create
+    @story = Story.new(story_params)
+
+    if @story.save
+      render json: @story
+    else
+      render :nothing => true, :status => 400
```

Comment:
same comment as above. this flow is a bit odd. you probably want to blow up if the save fails